.github/workflows/ci-app-accessibility-tests.yml

```diff
@@ -0,0 +1,66 @@
+name: CI App Accessibility Tests
+
+on:
+  workflow_call:
+    inputs:
+      service_endpoint_url:
```

Comment:
nit: could probably just call this `service_endpoint` to be consistent with the output name